packages/flutter_tools/schema/pubspec_yaml.json

```diff
@@ -50,6 +50,12 @@
                         "androidPackage": { "type": "string" }
                     }
                 },
+                "module": {
```

Comment:
This appears to be in conflict with the declaration above?

Reply:
Not yet.

Reply:
:) another go!